src/content/blog/astro-db-deep-dive.mdx

```diff
@@ -0,0 +1,77 @@
+
+Defeated, we put the idea on the back burner to focus on other things. It was Spring, 2023. 
+
+## Finding (and falling in love with) libSQL
+
+At this same time — halfway around the world and completely unknown to us — another team was working on this exact problem. That team was Turso and their solution was [libSQL](https://github.com/tursodatabase/libsql).
+
+libSQL is a fork of SQLite that introduces a collection of improvements to the runtime while maintaining compatibility with classic SQLite. libSQL featured a modern database client for JavaScript/TypeScript that avoided the native bindings and compilation steps that plagued the rest of the ecosystem. It could even run on [StackBlitz](https://stackblitz.com/) via WASM.
+
+Turso also offered hosting for libSQL databases with a specific focus on the kind of scale we needed (more on this in a bit). A vision was starting to come together for Astro DB, but it wasn't until December 2023 that all of the pieces would finally snap into place.
+
+## Designing a local database the Astro Way
+
+Astro DB gives you a fully-local libSQL database as soon as you start up your dev server. With our background as a static-site generator it was important that the database could be built from scratch on startup so that in the future it can power a content layer where the data is sourced from a variety of places, including the filesystem.
+
+When you run `astro dev`, Astro DB will:
+
+- Create an empty database at `.astro/data.db`
+- Read your schema from `db/config.ts`
+- Seed the database from `db/seed.ts`
+- Your database client is now ready
+
+The workflow largely mirrors the Content Collections workflow that Astro users already love. Importantly, the database itself (`data.db`) is *not* *persistent*. It's created new from scratch every time you start up the dev server. This gives you *simple, reproducible* one-off databases.
+
+Astro DB brings together the web framework, the schema, the seed file, and the database itself all into a single cohesive story. We even include an ORM for you: Drizzle. We selected Drizzle because it is a type-safe ORM that lets you get as close to the metal as you want, and is also pluggable which allowed us to add our own behaviors on top.
+
+## Going remote with a hosted database
+
+Astro DB includes a hosted libSQL database that you can connect to during local development and in production. Everything is managed for you through our Astro Studio platform. You can create a new database for your project in ~30 seconds.
+
+To achieve the kind of scale we knew we'd need on our platform, we partnered with Turso who maintain libSQL and operate the largest libSQL hosting platform. Their commitment to a “database per tenant” model was a perfect fit for our need to spin up hundreds of thousands of databases, all on-demand.
```

Comment:
I believe here the verbs should be in the singular(?), also, since we're using *on demand* as in *when demanded*, instead of as an adjective, it doesn't get hyphenated. 
```suggestion
To achieve the kind of scale we knew we'd need on our platform, we partnered with Turso who maintains libSQL and operates the largest libSQL hosting platform. Their commitment to a “database per tenant” model was a perfect fit for our need to spin up hundreds of thousands of databases, all on demand.
```